### ISSUE TITLE
How did you paired your device ?

### ISSUE THREAD

**Issue by jeanaaaa** [d03n3rfr1tz3/hass-divoom#1]
Hello,
Thanks for sharing your work.

I added the following configuration into configuration.yaml:
notify: 
  - name: pixoo_notif
    platform: divoom
    device_type: pixoo
    mac: "my:ma:c_:ad:dr:es"

and I am encountering the following error :
   2021-02-06 17:09:41 WARNING (SyncWorker_0) [custom_components.divoom.notify] Pixoo connection lost (errno = 107). Trying to reconnect for the 1 time.

I think that my device isn't automatically paired by your component.

How did you paired your device ?

**Comment by d03n3rfr1tz3** [comment]
Hi there and thank you for asking this.

You are basically right, I completely forgot to mention anything about that. Even worse, I also forgot to write that down in my own documentation of my Home Assistant. But I checked my rpi4 and found the commands I used, so I hopefully can fix that.

First I have to mention, that I'm running the Supervised installation of Home Assistant, therefore I have root access to the host system. Short version: Through SSH to the host system, I used "hcitool scan" to find the Divoom Pixoo and then used "sudo rfcomm connect hci0 my:ma:c_:ad:dr:es 1" to connect to it. 
Long version: I basically followed the instructions from https://www.pcsuggest.com/linux-bluetooth-setup-hcitool-bluez/

I'm not sure if there was anything more, but I'm pretty sure that from there on, the code I worked on had no connection issues. I hope it helps.
